its/src/main/java/org/jenkinsci/test/acceptance/po/ToolInstallation.java

```diff
@@ -132,6 +132,7 @@ public ConfigurablePageObject getPage() {
 
   public ToolInstallation installVersion(String version) {
     autoInstall.check();
+    sleep(1000);
```

Comment:
Is this intended for `master`, or just "temporary" code?

Reply:
Same as above